charts/k8s-monitoring/charts/feature-application-observability/README.md

```diff
@@ -40,6 +40,40 @@ Be sure perform actual integration testing in a live environment in the main [k8
 
 ## Values
 
+### Connectors: Grafana Cloud Host Info
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| connectors.grafanaCloudMetrics.enabled | bool | `true` | Generate host info metrics from telemetry data, used in Application Observability in Grafana Cloud. |
```

Comment:
@rlankfo Should this be on by default?

Reply:
@petewall I would say yes here given that the feature is intended for use with App O11y.